travis/apis.py

```diff
@@ -54,6 +56,18 @@ def __init__(self):
         self.ssh = os.environ.get(
             "WEBLATE_SSH", "ssh://user@webpage.com")
         self.tempdir = os.path.join(tempfile.gettempdir(), 'weblate_api')
+        self._ssh_keyscan()
+
+    def _ssh_keyscan(self):
+        cmd = "ssh-keyscan -p %s %s"
+        match = re.search(r'@(?P<port_host>[^/]+)', self.ssh)
+        if match:
+            port_host = match.groupdict()['port_host'].split(':')
+            host, port = ((port_host[0], 22) if len(port_host) == 1 else
+                          (port_host[0], port_host[1]))
+            cmd = cmd % (port, host)
+            with open(os.path.expanduser('~/.ssh/known_hosts'), 'w') as hosts:
```

Comment:
@JesusZapata I will always write the host, what sould happen with the others that could be already there?

Reply:
@ruiztulio 
I change the mode of open function to "+a"

Reply:
@JesusZapata @moylop260  If it ok for you to constantly add this key, then is ok for me (the detail I mentioned in the email)

Reply:
@ruiztulio @moylop260 
The method "_ssh_keyscan" only run one time

https://github.com/Vauxoo/maintainer-quality-tools/blob/5b0518f6e453f8dcb1d3c636f1b697d6934560c4/travis/travis_weblate.py#L40

Because the script "travis/travis_weblate.py" only run when the environment variable WEBLATE is present